files/image_config/kubernetes/kubernetes_master_entrance.sh

```diff
@@ -1,2 +1,23 @@
+#!/bin/bash
+
 # This script is for kubernetes master image usage
 # Will mount kubernetes master disk and execute kubernetes entrance script
+
+set -x
+
+# mount disk from host
+mount_point="/from_host"
+disk="/dev/sdb1"
```

Comment:
>sdb1 [](http://example.com/codeflow?start=11&length=4)

why hard-coded?

Reply:
We have a contract with VMM service, when VMM create master VM, it will attach a disk to the VM, and the disk has a certain location, and the name will be certain (/dev/sdb1). So I do hard-coded here.

Reply:
You are right, actually we can't ensure the device name is sda or sdb, I updated this part of code. We can specify the disk's location number when attach the disk to the SCSI controller of the VM. Inside VM, we can find the disk by disk path file, because the path file name includes the location number (lun-x). Thus, we can always mount the correct disk.

Reply:
Wow you might be good at hacking but your still stupid because I always find your online accounts...that's ok as you see I been on reporting things officially and you'll be out my phone off my internet and hopefully stop watching me on your 3D VM weirdo

Reply:
> >sdb1 [](http://example.com/codeflow?start=11&length=4)
> 
> 
> 
> why hard-coded? #Closed

Because he's hacking me with it. May you must be too